include/nccl_ofi_param.h

```diff
@@ -374,4 +374,9 @@ OFI_NCCL_PARAM_UINT(cm_num_rx_buffers, "CM_NUM_RX_BUFFERS", 32);
 OFI_NCCL_PARAM_VALUE_SET(PROGRESS_MODEL, (UNSPEC)(AUTO)(MANUAL))
 OFI_NCCL_PARAM(PROGRESS_MODEL, progress_model,  "PROGRESS_MODEL", PROGRESS_MODEL::UNSPEC)
 
+/*
+ * Override platform selection. Valid options: "AWS", "Default", or empty string for auto-detection.
+ */
+OFI_NCCL_PARAM(std::string, platform_override, "PLATFORM_OVERRIDE", "");
```

Comment:
Why does "override" make sense?  It doesn't match all the other option we have (like OFI_NCCL_PROGRESS_MODEL above).  This should just be OFI_NCCL_PLATFORM.